package.json

```diff
@@ -17,7 +17,7 @@
     "multihashes": "0.4.14",
     "node-fetch": "2.6.1",
     "priorityqueue": "1.0.0",
-    "spec-up": "0.9.79",
+    "spec-up": "0.9.82",
```

Comment:
glad we are updating this tool, would rather not see it as a dependency, given it has but one maintainer who is also chair and editor of the spec ; ) @csuwildcat .